servers/src/main/java/tachyon/worker/block/evictor/EvictorFactory.java

```diff
@@ -36,7 +36,7 @@ public static Evictor create(EvictorType evictorType, BlockMetadataManagerView v
     BlockMetadataManagerView managerView = Preconditions.checkNotNull(view);
     switch (evictorType) {
       case GREEDY:
-        return new GreedyEvictor(managerView);
+        return new GreedyEvictor();
```

Comment:
I'm not sure if this will work well with using reflection to create the evictor. But I guess we can fix that when that PR is opened...